vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1528,7 +1528,17 @@ def load_model(self) -> None:
         logger.info("Starting to load model %s...", self.model_config.model)
         with DeviceMemoryProfiler() as m:  # noqa: SIM117
             time_before_load = time.perf_counter()
-            self.model = get_model(vllm_config=self.vllm_config)
+            model_loader = get_model_loader(self.load_config)
+            if not hasattr(self, "model"):
+                self.model = model_loader.load_model(
```

Comment:
nit: shall we add some log here to tell we are going to load the model from scratch?